<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StagePrepare</class>
 <widget class="QMainWindow" name="StagePrepare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
        background-color: #ffff99;
}

QPushButton {
        background-color: #00cccc;
        color: #fff7e6 ;
        border-radius: 15;
}

QPushButton:hover {
        color: #fff7e6 ;
        background-color: #002e4d;
}

QPushButton:pressed {
        color: #fff7e6 ;
        background-color: #000066;
}

QLineEdit {
        background-color: #b3ecff;
        border-radius: 15;
        padding-left: 10;
        color: #0000b3;
        font-family : roboto;
}

QLabel {
        color: #005c99;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnReady">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ready</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUnready">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Unready</string>
    </property>
   </widget>
   <widget class="QLabel" name="text1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ROOM ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="text2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TEAMS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start !</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRoomID">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Room ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUserTeam">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>No data</string>
    </property>
   </widget>
   <widget class="QLabel" name="text3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PERMISSION</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUserStatus">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>No data</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGreet">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>15</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Greeting, username</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>340</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>340</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>3</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>3</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="textTeam1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>265</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Team 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="textTeam2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Team 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="textTeam3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>355</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Team 3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="textTeam5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Team 4: </string>
    </property>
   </widget>
   <widget class="Line" name="lineTeam1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>3</width>
      <height>195</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="lineTeam2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>660</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="lineTeam4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>445</y>
      <width>660</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="lineTeam3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>250</y>
      <width>3</width>
      <height>195</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChangeTeam">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Change Team</string>
    </property>
   </widget>
   <widget class="Line" name="lineTextGreeting">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>500</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelPlayerID">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>15</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PlayerID :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTeam1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>265</y>
      <width>550</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NaN , NaN , NaN</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTeam2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>550</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NaN , NaN , NaN</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTeam4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>550</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NaN , NaN , NaN</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTeam3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>355</y>
      <width>550</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NaN , NaN , NaN</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNotice">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>NOTICE: LOBBY UPDATED BELOW !</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnKick">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnKick {
        background-color: #ff3333;
}

QPushButton#btnKick:hover {
        color: #fff7e6 ;
        background-color: #002e4d;
}

QPushButton#btnKick:pressed {
        color: #fff7e6 ;
        background-color: #000066;
}</string>
    </property>
    <property name="text">
     <string>Kick</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionQuit_Lobby"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit_Lobby">
   <property name="text">
    <string>Quit Lobby</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
